client-templates/nexus/.env.sample

```
@@ -31,3 +31,6 @@ BROKER_HEALTHCHECK_PATH=/healthcheck
 BROKER_CLIENT_VALIDATION_URL=$BASE_NEXUS_URL/service/rest/v1/status/check
 
 BROKER_CLIENT_VALIDATION_JSON_DISABLED=true
+
+# Disable X-Forwarded-For headers so Nexus doesn't return npm tarball uri pointing to the broker server
+removeXForwardedHeaders=true
```

Recommend making it consistent with the other env vars.
```suggestion
REMOVE_X_FORWARDED_HEADERS=true
```